inc/VIPRCalculator.h

```diff
@@ -0,0 +1,73 @@
+#ifndef VIPRCALCULATOR_H
+#define VIPRCALCULATOR_H
+
+#include <VImageBaseAnalyzer.h>
```

Comment:
Same here, please use always `"VImageBaseAnalyzer.h"` (comment valid for all other Eventdisplay classes)